sdk/storage/CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ The environment variable **TEST_MODE** controls how the tests are running.
 - Else If TEST_MODE = "playback" (or if the TEST_MODE is not set or set to an invalid value),
   - Existing recordings are played back as responses to the HTTP requests in the tests
 
-Please refer to the [guidelines on Record and Playback](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/GUIDELINES.md) for more details.
+Please refer to the [guidelines on Record and Playback](https://github.com/Azure/azure-sdk-for-js/blob/c06170fbaf39e81496f8009a0da93d470d8c9f88/sdk/test-utils/recorder/GUIDELINES.md) for more details.
```

Comment:
Any reason to use a constant commit here?

Reply:
Thanks for the comment @EmmaZhu.
We are coming up with a new recorder based on the proxy tool built by the ENG SYS team.
Eventually, we'll be migrating all the services(tests) to use the new recorder.

Here's the migration guide - https://github.com/Azure/azure-sdk-for-js/pull/19210

The guides are different for the old and new recorders.
Since storage is still on the old recorder, I'm pointing it to the docs that are relevant to the old recorder.
Once storage packages are migrated to use the new proxy-tool based recorder, we'll update these links too.